helfi_features/helfi_profile_block/templates/profile-block.html.twig

```diff
@@ -0,0 +1,44 @@
+{% if logged_in %}
+  {% set link_attributes = {
+    'class': [
+      'profile__logout-link',
+    ],
+  } %}
+
+  {% set dropdown_id = 'profile-dropdown' %}
+
+  {% embed "@hdbt/misc/nav-toggle-button.twig" with {
+    modifier_class: 'nav-toggle--profile',
+    controls: dropdown_id,
+    anchor_target: '#profile',
+    js_target: 'js-profile-button',
+
+    open_label: display_name,
+    open_screenreader: 'Open profile information menu'|t({}, {'context': 'Profile information-menu open button text for screen readers'}),
```

Comment:
Should we contain some information here to the end screen reader end user that they're logged in as `display_name` like sighted users get?

Reply:
I added a text that tells screen reader user this information to the open_screenreader value.